test/stdtypes/concurrency_test.go

```diff
@@ -0,0 +1,51 @@
+package stdtypes
+
+import (
+	"io/ioutil"
+	"sync"
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+)
+
+func TestConcurrentTextMarshal(t *testing.T) {
```

Comment:
I just added it to the `test/stdtypes/Makefile` for now. Let me know if that works.